app/services/course_attribute_formatter_service.rb

```diff
@@ -0,0 +1,63 @@
+class CourseAttributeFormatterService
+  def initialize(name:, value:)
+    @name = name
+    @value = value
+  end
+
+  class << self
+    def call(**args)
+      new(args).call
+    end
+  end
+
+  def call
+    return value.tr("_", " ") if age_range?
```

Comment:
Could we not call `strip_underscores` instead?

Reply:
We could even call `age_range_value` which is what I meant to do 🤦‍♂ Bear with...